index.js

```diff
@@ -467,6 +467,9 @@ module.exports = {
           self.write(locale, page._url + '\n');
         }
       } else {
+        if (_.includes(self.excludeTypes, page.type)) {
```

Comment:
This will lose the children. You need to wrap the entire portion I pasted in slack in an "if" statement but not return... leave the "is this locale relevant" check outside the if though